src/CONST.ts

```diff
@@ -3050,13 +3050,6 @@ const CONST = {
      */
     MAX_OPTIONS_SELECTOR_PAGE_LENGTH: 500,
 
-    /**
-     * Performance test setup - run the same test multiple times to get a more accurate result
-     */
-    PERFORMANCE_TESTS: {
-        RUNS: 20,
```

Comment:
I assume 10 is default then when this was removed without substituting

Reply:
Yes, the default value for`runs` is 10.